src/test/java/org/folio/rest/api/HoldingsStorageTest.java

```diff
@@ -577,6 +614,82 @@ public void tenantIsRequiredForGettingAllHoldings()
   }
 
   @Test
+  public void updatingHoldingsUpdatesItemEffectiveCallNumber()
```

Comment:
I think it could be useful to have another test that demonstrates what happens when the call number is removed from the holdings record.

I think it could also be worth having a test that demonstrates that multiple items get updated when a holdings record changes, this could be an extension of this test or a new test.

Reply:
I've gotten everything working and updated the test to cover more than one item. Unfortunately I nothing happens when the call number is removed from a holdings, would you prefer I add it to this PR or should we open a new issue?

Reply:
I've updated the first test to include multiple items on a holdings and created a new test for when the call number is removed from the holdings.

Reply:
@rladdusaw Thanks